src/js/utils/DOMUtils.js

```diff
@@ -28,21 +29,19 @@ var DOMUtils = {
 
     var computeInnerBound = function (acc, key) {
       var val = parseInt(compstyle[key], 10);
-      if (_.isNaN(val)) {
+      if (isNaN(val)) {
```

Comment:
It doesn't seem like there is good support for it and Babel doesn't seem to have a polyfill for it: https://babeljs.io/repl/#?evaluate=true&lineWrap=false&presets=es2015%2Creact%2Cstage-2&experimental=false&loose=false&spec=true&playground=false&code=Number.isNaN('ajs')